### PR TITLE
Enable inductor CI for huggingface

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #87423
* __->__ #86792

Summary: Unit tests will be enabled after fixed in trunck. TorchBench and TIMM need
more setup and are coming later.

cc @jansel @lezcano @fdrocha @mlazos @soumith @voznesenskym @yanboliang

### CODE REVIEW

**Review by jansel — APPROVED** [review]

**Comment by jansel** [comment]
@desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor:
https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml

**Comment by desertfire** [comment]
> @desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor: https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml

This one is actually `TEST_CONFIG` based, and the plan is to have inductor unit tests running for all PRs. I will use the auto-labeler for integral tests later.

**Comment by desertfire** [comment]
> > @desertfire you can also add an auto-labeler rule to turn on these tests if people touch dynamo/inductor: https://github.com/pytorch/pytorch/blob/master/.github/labeler.yml
> 
> This one is actually `TEST_CONFIG` based, and the plan is to have inductor unit tests running for all PRs. I will use the auto-labeler for integral tests later.

With https://github.com/pytorch/pytorch/pull/87228, inductor unit tests do run for all PRs. I will update this PR to include auto-labeler and add more tests.

**Review by huydhn — APPROVED** [review]
LGTM!

File: .jenkins/pytorch/common_utils.sh [hunk 1]
```diff
@@ -170,6 +170,18 @@ function test_torch_deploy(){
...
+function install_huggingface() {
+  pip_install pandas
+  pip_install scipy
+  pip_install git+https://github.com/huggingface/transformers.git#egg=transformers
+}
+
+function install_timm() {
+  pip_install pandas
+  pip_install scipy
+  pip_install git+https://github.com/rwightman/pytorch-image-models
```
Comment (jansel) [inline .jenkins/pytorch/common_utils.sh:182]: Suggest pinning to a specific version

**Comment by desertfire** [comment]
[pytorchbot](https://github.com/pytorchbot) merge -g

**Pull request closed without merging**